### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by Rashmita-2000** [adiida/cnn-fruits-classification#1]
def create_rww_binary_crossentropy(fn_weight, fp_weight):
  """
  binary_crossentropy is a special case of categorical_crossentropy.  
  The "all other" column is imputed. For single-label, an implementation is provided.
  However, this loss function is not very useful in a multi-label environment. 
  It only supports k=1. 
  
  In future work, we will provide an implementation for one target, 
  one miscategorization. Note that this will
  require a tensor of at lease size (k,k,2,2), where, the first 2 
  represents T/F of target, the second 2 represents the FP/FN of label. 

  We may also create a fully expressive loss function, which would require some
  efficient representation of a tensor of size (k, k, 2^k, 2^k)
  
  """  
  def binary_loss_function(target, output):
    output = K.clip(output, K.epsilon(), 1 - K.epsilon()) 
    
    logs = K.log(output) # shape (m, 1)
    logs_1_sub = K.log(1-output) # shape (m, 1)

    return - K.mean(target * fn_weight * logs + 
                    (1-target) * fp_weight * logs_1_sub)

  return binary_loss_function

How do I implement this loss function in my model?

classifier = Sequential()
classifier.add(Conv2D(32, (3, 3), input_shape = (64, 64, 3), activation = 'relu'))
classifier.add(MaxPooling2D(pool_size = (2, 2)))
classifier.add(Flatten())
classifier.add(Dense(units = 128, activation = 'relu'))
classifier.add(Dense(units = 1, activation = 'sigmoid')) 
classifier.compile(optimizer = 'adam', loss = 'binary_crossentropy', metrics = ['accuracy']) #compiling the built CNN model
classifier.summary()

Instead of using binary cross entropy, I want to use the above loss function(create_rww_binary_crossentropy). Any idea how do I go about?
